src/views/download/download.jsx

```diff
@@ -0,0 +1,171 @@
+var React = require('react');
+var render = require('../../../lib/render.jsx');
+
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
+var FormattedMessage = require('react-intl').FormattedMessage;
+
+var Page = require('../../../components/page/www/page.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+
+require('./download.scss');
+
+var Download = React.createClass({
+    type: 'Download',
+    render: function () {
+        return (
+            <div className="download">
+                <TitleBanner className="masthead">
+                    <div className="inner">
+                        <h1 className="title-banner-h1">
+                            <FormattedMessage id='download.title' />
+                        </h1>
+                        <p className="title-banner-p intro">
+                            <FormattedMessage id='download.intro' />
+                        </p>
+                        <p className="title-banner-p intro">
+                            <FormattedHTMLMessage id='download.introMac' />
+                        </p>
+                    </div>
+                    <div className="band">
+                        <SubNavigation className="inner">
+                            <a href="#installation">
+                                <li>
+                                    <FormattedMessage id='download.installation' />
+                                </li>
+                            </a>
+                            <a href="#updates">
+                                <li>
+                                    <FormattedMessage id='download.updatesTitle' />
+                                </li>
+                            </a>
+                            <a href="#other">
+                                <li>
+                                    <FormattedMessage id='download.otherVersionsTitle' />
+                                </li>
+                            </a>
+                            <a href="#issues">
+                                <li>
+                                    <FormattedMessage id='download.knownIssuesTitle' />
+                                </li>
+                            </a>
+                        </SubNavigation>
+                    </div>
+                </TitleBanner>
+                <div className="inner">
+                    <section id="installation">
+                        <span className="nav-spacer"></span>
+                        <FlexRow className="three-col-row">
+                            <div class="installation column">
+                                <h3><FormattedMessage id='download.airTitle' /></h3>
+                                <p><FormattedHTMLMessage id='download.airBody' /></p>
+                                <p>
```

Comment:
Having the download statements be separate `<p>` tags like this makes them just as spaced out as they are from the text above:
<img width="471" alt="screen shot 2017-05-02 at 10 22 59 am" src="https://cloud.githubusercontent.com/assets/1815772/25622097/58e0d832-2f21-11e7-80e1-8e88eb65a905.png">
I think it might make sense to put these into a `ul` element, perhaps like:
```
<ul className="installation-downloads">
    <li className="installation-downloads-item"><a></a></li>
    <li className="installation-downloads-item"><a></a></li>
    <li className="installation-downloads-item"><a></a></li>
    <li className="installation-downloads-item"><a></a></li>
</ul>
```
And have the `ul` style not include bullet points, and have the `li` style not include padding or margins.